core-bundle/contao/drivers/DC_Table.php

```diff
@@ -1063,7 +1063,7 @@ protected function copyChilds($table, $insertID, $id, $parentId)
 		$ctable = $GLOBALS['TL_DCA'][$table]['config']['ctable'] ?? array();
 		$db = Database::getInstance();
 
-		if (!($GLOBALS['TL_DCA'][$table]['config']['ptable'] ?? null) && Input::get('childs') && $db->fieldExists('pid', $table) && $db->fieldExists('sorting', $table))
+		if (Input::get('childs') && $db->fieldExists('pid', $table) && $db->fieldExists('sorting', $table))
```

Comment:
Could it be, that instead of this change that we need to set `$GLOBALS['TL_DCA']['tl_content']['config']['ctable']` to `array('tl_content')` in *contao/dca/tl_content.php* ?

Reply:
No, I think pointing both `ptable` and `ctable` to the same table is dangerous and will cause problems.

Can you explain why you think that we need the `!($GLOBALS['TL_DCA'][$table]['config']['ptable'] ?? null)` check? It seems like an unnecessary limitation to me. Any table with a `pid` and a `sorting` column can be copied with children, regardless of whether it has a parent table or not.

Reply:
Looks like this caused a bug: #7422 
As we have `tl_content` as `ctable` now in https://github.com/contao/contao/blob/0b009b91b4186a32e91ca5f762df5b6a6e4ee375/core-bundle/contao/dca/tl_content.php#L37 should we revert this change?